src/middlewares/authMiddleware.js

```diff
@@ -0,0 +1,21 @@
+import { jwtService } from "../services/jwt.service.js";
+
+export const authMiddleware = (req, res, next) => {
+  const authorization = req.headers['authorization'] || '';
+  const [,token] = authorization.split(' ');
+
+  if (!authorization || !token) {
+    res.status(401).send('Token is not valid');
+    return;
+  }
+
+  const userData = jwtService.verify(token);
+  console.log(userData);
```

Comment:
Logging user data with `console.log(userData);` can expose sensitive information in the server logs. Consider removing this line or ensuring that sensitive data is not logged.